# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dosbox/gcc46.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ru dosbox-0.74.orig/include/dos_inc.h dosbox-0.74/include/dos_inc.h
--- dosbox-0.74.orig/include/dos_inc.h	2010-05-10 12:43:54.000000000 -0500
+++ dosbox-0.74/include/dos_inc.h	2012-03-10 15:19:47.000000000 -0600
@@ -21,6 +21,8 @@
 #ifndef DOSBOX_DOS_INC_H
 #define DOSBOX_DOS_INC_H
 
+#include <stddef.h>
+
 #ifndef DOSBOX_DOS_SYSTEM_H
 #include "dos_system.h"
 #endif
diff -ru dosbox-0.74.orig/src/cpu/cpu.cpp dosbox-0.74/src/cpu/cpu.cpp
--- dosbox-0.74.orig/src/cpu/cpu.cpp	2010-05-12 04:57:31.000000000 -0500
+++ dosbox-0.74/src/cpu/cpu.cpp	2012-03-10 15:17:54.000000000 -0600
@@ -19,6 +19,7 @@
 /* $Id: cpu.cpp,v 1.116 2009-03-16 18:10:08 c2woody Exp $ */
 
 #include <assert.h>
+#include <stddef.h>
 #include <sstream>
 #include "dosbox.h"
 #include "cpu.h"
